sensor/kubernetes/listener/resources/complianceoperator/dispatchers/complianceoperatorscansettingbindings.go

```diff
@@ -32,6 +62,28 @@ func (c *ScanSettingBindings) ProcessEvent(obj, _ interface{}, action central.Re
 		log.Errorf("error converting unstructured to compliance scan setting binding result: %v", err)
 		return nil
 	}
+	id := string(scanSettingBindings.UID)
```

Comment:
Going with the 2 pipeline approach if compliance is enabled, then we need to send both V1 and V2 messages.  

I've been doing it like this:

```
events := []*central.SensorEvent{...}

if centralcaps.Has(centralsensor.ComplianceV2Integrations) {
   events = append(events, &central.SensorEvent{...})
}

return component.NewEvent(events...)
```